src/widgets/cards/summary-card.css

```diff
@@ -6,7 +6,6 @@
 }
 
 :global(.omrs-breakpoint-gt-tablet) .card {
-  width: 21.75rem;
```

Comment:
If we remove this, we will no longer have the ability to have two columns of widgets. I will try to think of how to redo this but please do so as well.